docs/quality-assurance/test-policy

```diff
@@ -0,0 +1,62 @@
+# Purpose and Scope
+
+The purpose of this document is to communicate the policy for UKHO for testing of new software, updates, hardware and other related changes.
+
+## Why do we Test?
+
+* To build confidence in our systems
+* Detect defects
+* Generate information
+* Manage quality risks
+
+## How do we measure the effectiveness of testing?
+
+Delivery to customers of software and hardware that provides business value.
+
+## UKHO Long Term Testing Goals
+
+* To move to a DevOps Model
+* Test Automation provides the bulk of testing functionality
+* High Value manual testing is used
+* Testing across UKHO is done within an Agile framework
+
+## Who Tests?
+
+Test Engineering line manages a group of test engineers, ranging from junior to lead people; they provide testing capability across most of UKHO where testing is carried out within Scrum teams.
+
+Outside of the Scrum teams other people do test, for instance within Operations testing of hardware is carried out by the relevant SME, like network engineers carry out network testing. In some non-Agile teams, Business Analysts also carry out testing.
+
+## Who Manages Testing?
+
+The Head of Test line manages all test engineers, but test engineers are tasked managed by the overall technology program in conjunction with the Head of Test.
+
+Other areas of the business carry out various testing activities, but these people will not necessarily be professional software testers, they are line managed by their relevant program.
+
+## Roles and Responsibilities of Test Engineering
+
+This will only deal with the roles within Test Engineering.
+
+## In Scope
+
+* All software applications
+* Any changes to database structures
+* Third-party software
+* Cloud-based applications
+
+## The Testing Life-Cycle
+
+UKHO follows the Disciplined Agile Delivery (DAD) and testing forms part of that life-cycle process which is collectively referred to as the Agile Delivery Governance Framework (ADGF).
+
+Testing will be carried out at all stages of the development of software and products.
+
+There are sub-policies that cover the Agile and Non-Agile methods as well as security, penetration , load and performance testing.
+
+The test guidance within the ADGF provides guidance/support for all teams that will carry out testing across UKHO.
+
+## Testing Standards
+
+Our baseline of standards are the BS ISO/IEC/IEEE 29119.
```

Comment:
Might be worth mentioning the Must/Expected/Recommended standards and how the test community agrees them by consensus

Reply:
Could also link to the Test Standards page already on GitHub

Reply:
So test standards fit based on ISTQB into Test Strategy which is the next doc I want to write